Downgraded Android SDK to work with CircleCI

maxmousee opened the pull request.

maxmousee committed:
Downgraded Android SDK to work with CircleCI

maxmousee committed:
Merge with master

maxmousee committed:
Added dependencies for circleCI

maxmousee committed:
Updated libraries

maxmousee committed:
Rolled back view api calls

maxmousee committed:
Test circleCI license fix

maxmousee committed:
Updated libs

maxmousee committed:
Merge with master

diegolmendonca reviewed (commented):
Natan, can you perform a pull? It looks like some files are outdated.. for instance,  AttendeeResponseService

maxmousee commented:
Done @diegolmendonca (: 
I think now it's ready to be merged

maxmousee committed:
Fixed typos

diegolmendonca committed:
Update AttendeeResponseService.kt

I dont know why, but this is reverting back to old version. I am updating it, so that merge does not break

diegolmendonca committed:
Update strings.xml

diegolmendonca committed:
Update strings.xml

diegolmendonca committed:
Update strings.xml
